lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -45,7 +45,8 @@ define((require, exports, module) ->
       ).join("\n")
       startIndex = session.doc.positionToIndex(start, start.row)
       endIndex = session.doc.positionToIndex(end, start.row) + 1
-      content = joinedLines.substring(startIndex, endIndex - 1)
+      content = joinedLines.substring(startIndex, endIndex - 1).replace(/%[^\n]*/g, "")
```

Comment:
will it remove escaped percents too?

Reply:
This code is outdated now. But if you're wondering about this version, no, it wasn't able to detect escaped percent signs. It's one of the reasons I decided to offload filtering of comments and labels to `PapeeriaLatexHighlightRules`.